### ISSUE TITLE
Support Sedulight

### ISSUE THREAD

**Issue by FloEdelmann** [OpenLightingProject/ola#1187]
[Sedulight](http://www.insaneware.de/epages/61714203.sf/de_DE/?ObjectPath=/Shops/61714203/Products/sedu) is an LED strip controller that gets its data via USB (possibly very similar to KarateLight). The protocol used is [MiniDMX](https://www.dzionsko.de/pmwiki.php?n=MiniDMX.Protokoll) (german). I guess it should be relatively easy to implement.

**Comment by peternewman** [comment]
I was going to say how's your German @FloEdelmann, but I see you're in Germany, so undoubtedly better than mine. Saying that, even I could read enough to see it appeared to be a fairly trivial protocol to implement.

If you really want to gold-plate it, you could add options to offset the data it sends within the universe (on the 96 and 256 slot versions, so you can get a few of them to cover a whole universe of data.

Have you got one of the Sedulight devices, that's the normal stumbling block on device support?

**Comment by FloEdelmann** [comment]
Yep, we got one at home to use it as an Ambilight clone but couldn't get it working because the Enigmalight plugin is not running on our set-top box. But maybe we can at least make some DMX use of it :)

Offsetting is a good idea if the box does report a serial number so we can distinguish different boxes. I can imagine that this (relatively cheap) box does not, though.

And yes, my German is quite good ;)
